spec/models/friendship_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe Friendship, type: :model do
+  describe 'associations' do
+    it { should belong_to(:user).class_name('User') }
```

Comment:
- [ ] Add Unit and Integration tests for your app. Ensure to cater for all models